docs/how-to-guides/themes/create-block-theme.md

```diff
@@ -238,12 +238,12 @@ Edit and save the template in the same way as in the site editor.
 Templates and template parts that have been created or edited in the site editor or template editing mode
 are saved to the database as custom post types. To export them as theme files, follow these steps:
 
-- In the site editor, open the **More tools and options** menu.
-- Select the **Export** option to download a zip file containing the files. Unpack the files.
-- Copy the updated `index.html` file from `theme/templates/` to your theme's `templates` folder.
-- Copy template part one and two from `theme/parts/` to your theme's `parts` folder.
-- Rename the template parts to `header.html` and `footer.html`, respectively.
-- Open `index.html` and update the template part slugs in the block markup.
+-   In the site editor, open the **More tools and options** menu.
+-   Select the **Export** option to download a zip file containing the files. Unpack the files.
+-   Copy the updated `index.html` file from `theme/templates/` to your theme's `templates` folder.
+-   Copy template part one and two from `theme/parts/` to your theme's `parts` folder.
+-   Rename the template parts to `header.html` and `footer.html`, respectively.
+-   Open `index.html` and update the template part slugs in the block markup.
 
 Saved templates have precedence over theme files. To use the updated theme files, go to **Appearance > Templates** and
 **Appearance > Template parts** and delete the saved templates.
```

Comment:
Appearance > templates was removed so this is not correct anymore.

I think that most of this passage can be removed and a link to the new site editor article can be added. https://wordpress.org/support/article/site-editor/

```suggestion
Saved templates have precedence over theme files.

To learn more about the Site Editor, see the [support article](https://wordpress.org/support/article/site-editor/)
```